react/features/base/config/reducer.ts

```diff
@@ -293,6 +293,48 @@ function _translateInterfaceConfig(oldValue: IConfig) {
         }
     }
 
+    // if we have `deeplinking` defined, ignore deprecated values. Otherwise, compose the config.
+    if (!oldValue.deeplinking) {
+        const disabled = Boolean(oldValue.disableDeepLinking || typeof interfaceConfig !== 'object');
```

Comment:
What are you trying to test with the second part of the OR?

Reply:
Basically if the config does not contain the new `deeplinking` and interfaceConfig is undefined - we have no way of getting the deeplinking configs, so we disable.

Reply:
We should use the defaults, the current values are the defaults.

Reply:
Added function for setting defaults.